components/layout/variables.styl

```diff
@@ -98,7 +98,9 @@ privacy-light-color(prefix, attr)
   colors = 'closed' closed, 'public' public, 'private' private
   for value in colors
     {prefix + value[0]}
-      {attr} alpha(value[1], 0.1)
+      if attr == border-color
+        border 2px solid
+      {attr} alpha(value[1], 0.3)
```

Comment:
*much* easier... i see nothing wrong with that code. those mystery meat functions in the stylesheets are freaking me out.